postfix/tests/test_unit.py

```diff
@@ -0,0 +1,30 @@
+# (C) Datadog, Inc. 2019
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+import mock
+
+from datadog_checks.postfix import PostfixCheck
+
+MOCK_VERSION = '1.3.1'
+
+
+@mock.patch(
+    'datadog_checks.postfix.postfix.get_subprocess_output',
+    return_value=('mail_version = {}'.format(MOCK_VERSION), None, None),
+)
+def test_collect_metadata(aggregator, datadog_agent):
```

Comment:
How does integration/e2e works for postfix? Can we also run a similar test against a real container?

Reply:
There is currently no integration/e2e with actual `postfix` installed. 

Let check if I can add a integration/e2e for that.

Reply:
I added this comment `# TODO: Migrate this test as e2e test when it's possible to retrieve the metadata from the Agent`

I added support for e2e here https://github.com/DataDog/integrations-core/pull/4948
So, we can test in e2e when we can retrieve metadata from the agent.

It seems not possible to implement easily the test as integration since you need postfix to be installed on the same machine.